shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/properties/ConfigurationPropertyKey.java

```diff
@@ -116,7 +116,7 @@
     /**
      * Whether enable show process list.
      */
-    SHOW_PROCESS_LIST_ENABLED("show-process-list-enabled", String.valueOf(false), boolean.class),
+    SHOW_PROCESS_LIST_ENABLED("show-process-list-enabled", String.valueOf(Boolean.FALSE), boolean.class),
```

Comment:
Hi why do we use `Boolean.FALSE`? I suppose the third parameter defines the type of this property and will do conversion somehow. In that way, do you think we need to do such changes?

Reply:
@LeeGuoPing could you give more explanation?

Reply:
@tristaZero Other place of this class, use `String.valueOf(Boolean.FALSE)`. I think it is more better use the same way.
 
![image](https://user-images.githubusercontent.com/35389145/130019623-9a337f17-5ea0-4db2-8182-a2c3a7247251.png)